committing.rst

```diff
@@ -208,14 +208,14 @@ repositories means you have to be more careful with your workflow:
   dedicated to maintenance of the work before the work gets integrated in the
   main repository.
 
-  An exception to this rule: you can make a quick edit through the web UI of
-  GitHub, in which case the branch you create can exist for less than 24 hours.
-  This exception should not be abused and be left only for very simple changes.
-
 * You should not commit directly into the ``master`` branch, or any of the
   maintenance branches (currently ``2.7`` or ``3.6``).
   You should commit against your own feature branch, and create a pull request.
 
+* For a small change, you can make a quick edit through the web UI of
+  GitHub. You will still create a branch on your fork, and the change will
+  go through the PR process.
```

Comment:
Unfortunately, if you have merge access on the target repo, GH doesn't provide a way to request that the PR branch be created on your fork (if it did, we'd just say to use that option, and wouldn't need the policy exception).

Instead, the "Create a PR for this change" option only lets you create the branch directly in the repo you're editing.

Reply:
Good point. I never realized that is what it did. I only use it for quick typos and delete the branch right after.

I'm inclined to remove the entire paragraph as most changes can go through the PR process even if the dev self merges the PR for typos and such.